fast_bert/learner_cls.py

```diff
@@ -3,6 +3,8 @@
 from .learner_util import Learner
 from torch import nn
 from typing import List
+import torch.cuda as cutorch
```

Comment:
Not familiar, ca sert a quoi?

Reply:
ah that was just to get cuda memory usage to do batch size optimization